v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs

```diff
@@ -1,98 +1,99 @@
 ﻿// --------------------------------------------------------------------------------------------------------------------
-// <copyright file="EventController.cs" company="AdminCore">
+// <copyright file="HolidayController.cs" company="AdminCore">
 //   AdminCore
 // </copyright>
 // <summary>
-//   Defines the EventController type.
+//   Defines the HolidayController type.
 // </summary>
 // --------------------------------------------------------------------------------------------------------------------
 
-using System.Collections.Generic;
+using System;
 using AdminCore.Common.Interfaces;
-using AdminCore.WebApi.Models.Event;
 using AutoMapper;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using AdminCore.WebApi.Models.Event;
 
 namespace AdminCore.WebApi.Controllers
 {
   [Authorize]
   [Route("[controller]")]
   [ApiController]
-  public class EventController : ControllerBase
+  public class HolidayController : ControllerBase
   {
     private readonly IMapper _mapper;
 
     private readonly IEventService _eventService;
 
-    public EventController(IEventService eventService, IMapper mapper)
+    public HolidayController(IEventService holidayService, IMapper mapper)
```

Comment:
It's still the event service

Reply:
So the Interface also needs to be IHolidayService?